src/afl-common.c

```diff
@@ -108,7 +108,7 @@ void set_sanitizer_defaults() {
     if (!have_san_options) { strcpy(buf, default_options); }
     if (have_asan_options) {
 
-      if (NULL != strstr(have_asan_options, "detect_leaks=0")) {
+      if (NULL != strstr(have_asan_options, "detect_leaks=false")) {
```

Comment:
why this? it is detect_leaks=0 and false does not exist according to the documention https://github.com/google/sanitizers/wiki/AddressSanitizerLeakSanitizer

Reply:
It's set here https://github.com/AFLplusplus/AFLplusplus/blob/dev/src/afl-fuzz.c#L1938 and it judging from this [documentation](https://github.com/google/sanitizers/wiki/SanitizerCommonFlags) it can be "true" or "false". I guess in the end it doesn't make a difference for ASAN itself.

Reply:
So maybe use an integer value everywhere, or handle both cases? Also in the Frida mode documentation it is detect_leaks=false is mentioned.